notebooks/article_1.py

```diff
@@ -0,0 +1,1201 @@
+# 
+# Finally, startups secure funding from to types of source:
+# - **Angel investors** are high net worth individuals who provide personal capital for investment. - **venture capital firms** are corporate entities specializing in investing in startups.
+# 
+# Let’s get in to the preliminary steps. 
+
+# ## Data Exploration
+
+# Data exploration serves as the first step in data analysis, involving the examination of datasets to identify patterns and detect anomalies.
+# 
+# The following libraries provide the necessary tools and functions for data manipulation, analysis, and visualization.
+
+# In[2]:
+
+
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import plotly.graph_objects as go
+import plotly.express as px
+from scipy.stats import zscore
+
+
+# Now, we load the data into a [Pandas](https://pandas.pydata.org/) DataFrame.
+
+# In[3]:
+
+
+file_path = os.path.join('..', 'saved_model', 'startup.csv')
+data_df = pd.read_csv(file_path, encoding="ISO-8859-1")
+
+
+# In[4]:
+
+
+data_df.head(2)
+
+
+# Next, let's focus on understanding the significance of the Target Variable.
+
+# The `status` variable is the most important feature in this dataset. This binary variable indicates whether a startup has been acquired or closed.
+# 
+# When analyzing the variable, it shows that 597 startups in our dataset were acquired, while 326 startups closed down.
+
+# In[5]:
+
+
+data_df['status'].head(2)
+
+
+# ## Data Pre-processing
+
+# The pre-processing step involves cleaning and organizing the raw data.
+# 
+# We'll accomplish following these steps:
+# 
+# - The **Data cleaning** step consists on identifying and correcting data errors, such as typos, duplicates, or incorrect entries. It is essential for maintaining data integrity. We'll focus on handling:
+# - - The irrelevant features
+# - - The missing values
+# - - The negative values where it shouldn't be so.
+# 
+# - The **Data transformation** step modifies data to re-organize them into a format appropriate for performing analysis. These are the common transformatio that we will perform:
+# - - **Normalization** and **standardization**: These techniques adjust data to a common scale, making it easier to compare them.
+# - - **Encoding categorical data**: Transforming categorical variables into numerical formats.
+
+# ### Handle Irrelevant Features
+
+# Irrelevant feature refers to a data attribute or variable that does not provide valuable or meaningful information for the task at hand. Here are some reasons why removing them is beneficial:
+# 
+# - Eliminating irrelevant variables simplifies the analysis process.
+# - It ensures focus on meaningful factors by reducing clutter and improving clarity in data exploration.
+# 
+# To accomplish this, we will implement the following method.
```

Comment:
Say here that we will drop the following columns:

Then state that we added a comment for the reason to drop each one.

Finally, add a space after #.
#Irrelevant data -> # Irrelevant data